src/AutomatedCar/Models/AutomatedCar.cs

```diff
@@ -13,13 +15,42 @@ public class AutomatedCar : Car, IAutomatedCar
         private const int MIN_PEDAL_POSITION = 0;
         private const int MAX_PEDAL_POSITION = 100;
         private const double PEDAL_INPUT_MULTIPLIER = 0.01;
-        private const double DRAG = 0.006; // This limits the top speed to 166 km/h
+        private const double DRAG = 0.006; // This limits the top speed to 166 km/h
+        private const int IDLE_RPM = 800;
+        private const int NEUTRAL_RPM_MULTIPLIER = 100;
+        private const int PIXEL_METER_RATIO = 50;
+
+        private const int CAR_MASS = 1500;
+        private const double TRANSMISSION_EFFICIENCY = 0.7;
+        private const double WHEEL_RADIUS = 0.34;
+        private const double DIFFERENTIAL_RATIO = 3.42;
 
         private int gasPedalPosition;
-        private int brakePedalPosition;
+        private int brakePedalPosition;
+        private int revolution;
+        private int innerGear = 1;      //manually set until inner gearbox is implemented
+        private int engineForce;
+        private int currentTorque;
 
         private VirtualFunctionBus virtualFunctionBus;
-        private ICollection<ISensor> sensors;
+        private ICollection<ISensor> sensors;
+
+        private Dictionary<int, int> TorqueLookupTable = new Dictionary<int, int>()

```

Comment:
We are getting overcrowed here. Can we outsorce these config parameters somehow?

Reply:
I have moved all car-related constans to a static config class.